src/test/java/com/gooddata/notification/ChannelConfigurationTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2004-2017, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.notification;
+
+import static com.gooddata.util.ResourceUtils.readObjectFromResource;
+import static net.javacrumbs.jsonunit.JsonMatchers.jsonEquals;
+import static net.javacrumbs.jsonunit.core.util.ResourceUtils.resource;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.testng.annotations.Test;
+
+public class ChannelConfigurationTest {
+
+    @Test
+    public void testDeserialization() throws Exception {
+        final ChannelConfiguration channelConfiguration = readObjectFromResource("/notification/channelConfiguration.json", ChannelConfiguration.class);
+
+        assertThat(channelConfiguration.getMeta(), is(notNullValue()));
+        assertThat(channelConfiguration.getMeta().getUri(), is("/gdc/account/profile/876ec68f5630b38de65852ed5d6236ff/channelConfigurations/59dca62e60b2c601f3c72e18"));
+        assertThat(channelConfiguration.getMeta().getTitle(), is("test channel"));
+        assertThat(channelConfiguration.getConfiguration(), is(notNullValue()));
```

Comment:
I suggest testing also the `configuration` field... It should be of type `EmailConfiguration` and contain specified email address.

Reply:
done